js/classes/asset_manager.js

```diff
@@ -75,5 +113,64 @@ class AssetManager {
     clearCache() {
         this.cache = [];
     };
+
+    /**
+     * This method plays the audio associated with the given path.
+     * @param {string} path The filepath of the audio you are trying to play.
+     * @param {number} volume The volume to which you want to set the audio.
+     * @param {boolean} loop True if you want the audio to loop, false otherwise.
+     */
+    playAudio(path, volume, loop) {
+        loop ?? (loop = false);
```

Comment:
Is line 124 correct use of the ?? operator?
I'm not certain if your way is proper or not, but i think i would more likely do something like "loop = loop ?? false"

Reply:
Good point, I'm replacing that with a line with this: `loop = loop || false;`